### ISSUE TITLE
Warning: Missing js property. Use --force to continue.

### ISSUE THREAD

**Issue by reintroducing** [gmarty/grunt-closure-compiler#29]
I'm trying to run a build and I keep seeing this warning.  Here is my task:

```
'closure-compiler': {
            frontend: {
                closurePath: '/tools',
                js: '/en/2014/js/award_tiles.js',
                jsOutputFile: '/en/2014/js/vlp.min.js',
                options: {
                    compilation_level: 'SIMPLE_OPTIMIZATIONS'
                }
            }
        }
```

Please advise.

**Comment by gmarty** [comment]
Use an absolute path for the JS file location or use '../' for a relative path.

**Comment by reintroducing** [comment]
yeah, apologies, i realized that about 10 minutes ago :)  Thanks.

**Comment by reintroducing** [comment]
apologies for reopening this but I finally got back around to trying this out.  here is my setup: 

```
'closure-compiler': {
            closurePath: '../../tools/google-closure-compiler',
            js: 'js/award_tiles.js',
            jsOutputFile: 'js/vlp.min.js'
        }
```

my Gruntfile is in work/en/2014, the js files are pathed from there appropriately, the compiler.jar file is in work/tools/google-closure-compiler.  I'm getting this error:

$ grunt closure-compiler                                                                                                      ± ‽
Running "closure-compiler:closurePath" (closure-compiler) task

> > /!\ Set an environment variable called CLOSURE_PATH or the build parameterclosurePath and
> > make it point to your root install of Closure Compiler.

In my case, what am I pathing wrong for closurePath?

**Comment by reintroducing** [comment]
OK, so I was able to get around this by just setting CLOSURE_PATH up and am not getting this error anymore.  That being said, I now have a new error:

$ grunt closure-compiler                                                                                                    ⏎ ± ‽
Running "closure-compiler:js" (closure-compiler) task
Warning: Cannot call method 'indexOf' of undefined Use --force to continue.

Any idea why i'm getting this?

**Comment by reintroducing** [comment]
Aaaaaand nevermind, got this working, had to wrap my task inside of a frontend: {} object like my original post had (which followed what you have in the readme).  My mistake. Apologies.
